airflow/operators/check_operator.py

```diff
@@ -137,6 +137,7 @@ def __init__(
     def execute(self, context=None):
         logging.info('Executing SQL check: ' + self.sql)
         records = self.get_db_hook().get_first(self.sql)
+        logging.info("Record: " + str(records))
```

Comment:
Is there any chance that this is a nontrivial amount of data we're shoving in the logs?

Reply:
This operation is expected to return only one row